app/models/unit.rb

```diff
@@ -38,4 +39,5 @@ class Unit < ApplicationRecord
   scope :unit_status_id, ->(id) { where(unit_status_id: id) }
   scope :unit_instruction_type_id, ->(id) { where(unit_instruction_type_id: id) }
   scope :unit_instruction_language_id, ->(id) { where(unit_instruction_language_id: id) }
+  scope :committees, -> { where(unit_type: UnitType.find_by(name: 'Kurul / Komisyon')) }
```

Comment:
Hali hazırda böyle bir UnitType yok. Bu branch merge edildiğinde geliştiriciler böyle bir UnitType oluşturmaları gerektiğini nereden bilecekler? Üç senaryoyu da kapsaman lazım:

1) Şuan kodun üzerinde çalışan geliştiriciler (PR gövdesinde ilgili alana ne yapılması gerekiyorsa eklemelisin)
2) Uygulamayı sıfırdan kuracak geliştiriciler (seed'e de eklemen lazım).
3) Dokku'da ne olacak. (`lib/tasks/post_deploy` altına bir task oluşturman lazım).

Ek olarak name ile `find_by` yapmak yerine Kurul / Komisyon'a sabit bir `code` tanımlayıp onu kullasan daha güzel olmaz mı? Yarın biri gelip sırf gözüne daha güzel göründüğü için `/`'tan önceki ve sonraki boşlukları silip uygulamayı bozabilir.